pkg/sqlite/load.go

```diff
@@ -893,6 +893,7 @@ func (s *sqlLoader) addPackageProperty(tx *sql.Tx, bundleName, pkg, version, bun
 }
 
 func (s *sqlLoader) addBundleProperties(tx *sql.Tx, bundle *registry.Bundle) error {
+	// FIXME: bundle.cache() seems like the proper place for this to be generated. Let's investigate what that looks like at some point.
```

Comment:
Is there a reason this approach wasn't taken the first time around? It seems much nicer to encapsulate this logic for all storage implementations than to rewrite it for each.